docs/swagger.yaml

```diff
@@ -517,6 +508,37 @@ components:
                   type: string
                   description: How long the average rental period is
                   example: 3 weeks
+            prepaidServices:
+              type: object
+              description: Only required if the sale unit will do prepaid services
+              required:
+                - turnoverShare
+                - prepurchaseTime
+              properties:
+                turnoverShare:
+                  type: string
+                  description: Estimated turnover share of prepaid services sales in percentage
+                  example: about 25%
```

Comment:
Should we make this a number instead of string? If we make it a number _maybe_ we can automate it more (at some point) than if we use string, since strings have to be parsed/understood, whereas numbers are "ready to use". If we go for numbers we must of course explain that it does not have to be exact.

Reply:
I would leave it as string. As the partner or merchant can give text to describe their scenario as well. In the frontend in merchant portal it is a free text and not enforced as number. So we can leave it as is here, imo.